Makefile

```diff
@@ -20,11 +20,11 @@ iso_creation: init
 .PHONY: centos_iso
 centos_iso: KICKSTART_FILE=centos-7-minimal.ks
 centos_iso: KICKSTART_TEMPLATE=centos-7-minimal.template
-centos_iso: ISO_NAME=minishift-centos.iso
+centos_iso: ISO_NAME=minishift-centos
 centos_iso: iso_creation
 
 .PHONY: rhel_iso
 rhel_iso: KICKSTART_FILE=rhel-7-minimal.ks
 rhel_iso: KICKSTART_TEMPLATE=rhel-7-minimal.template
-rhel_iso: ISO_NAME=minishift-rhel.iso
+rhel_iso: ISO_NAME=minishift-rhel
```

Comment:
@LalatenduMohanty even downstream product *CDK* will have minishift as starting point `minishift start` not `cdk start` right? I think it make sense to have `minishift-rhel` as image name.